views/account/settings/index.ejs

```diff
@@ -136,17 +156,31 @@
                   name="theme"
                   class="form-control"
                   id="editTheme"
+                  aria-label="theme"
                 ></select>
               </div>
             </div>
           </div>
+          <br />
```

Comment:
2 things: 

- I'd like to see if there is a way to have the confirm prompt not within the onclick attribute
- I think the button should go after the save changes and cancel changes buttons

Reply:
- i was looking into it for like hours and i still don't understand why it's just not pulling from the main.js file. i can still keep working on it because i really want all logic/JS to be in the main.js file. 

- got it, im fine with that too! will do it right away :)

Reply:
You can fix the button placement, I'll look into the button.